### DISCUSSION TITLE
Video Download Helper

### DISCUSSION THREAD

**Discussion by ga6riela** [aclap-dev/video-downloadhelper#417] [License & Purchase requests]
### What's your issue?

Just wondering if the "Video Download Helper" is the same as "Video Conversion" to download videos.

I see on the site it's saying it's a one time license fee without expiration but it's been a year and I can't download videos anymore. It's suggesting me to the page to purchase it again for $28.50. 

Would like to keep downloading videos. If I spend the $28.50 will this give me access again to downloading the videos?

Is it a yearly fee to use the downloader?

Thanks

![image](https://github.com/aclap-dev/video-downloadhelper/assets/167653582/ab91c436-f9de-4615-bbee-06353583dee5)

### The 10 first digits (only 10) of your license number or email address

ga6riela@live.ca

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Licenses are for life, there is no additional fee.
I sent you an email.
jerome
